cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityUserOperator.java

```diff
@@ -34,6 +34,9 @@
     protected static final int HEALTHCHECK_PORT = 8081;
     protected static final String HEALTHCHECK_PORT_NAME = "healthcheck";
 
+    public final static int DEFAULT_CERTS_VALIDITY_DAYS = 365;
```

Comment:
Shoudn't the defaults be set in the `api`? Wasn't that how we were doing it lately?